HLTrigger/Egamma/plugins/HLTScoutingEgammaProducer.cc

```diff
@@ -205,15 +207,17 @@ void HLTScoutingEgammaProducer::produce(edm::StreamID sid, edm::Event& iEvent, e
     float sMin = moments.sMin;
     float sMaj = moments.sMaj;
 
-    unsigned int seedId = (*SCseed).seed();
+    uint32_t seedId = (*SCseed).seed();
 
     std::vector<DetId> mDetIds = EcalClusterTools::matrixDetId((topology), (*SCseed).seed(), rechitMatrixSize);
 
     int detSize = mDetIds.size();
+    std::vector<uint32_t> mDetIdIds(detSize, 0);
     std::vector<float> mEnergies(detSize, 0.);
     std::vector<float> mTimes(detSize, 0.);
 
     for (int i = 0; i < detSize; i++) {
+      mDetIdIds[i] = mDetIds.at(i);
```

Comment:
Just to note that this (and below) could be
```suggestion
      mDetIdIds[i] = mDetIds[i];
```
given that the loop is bounded by `mDetIds.size()`. `[]` instead of `at()` would be a little bit cheaper.

Reply:
Will do!